tests/test_ranges.py

```diff
@@ -408,6 +424,22 @@ def test_can_handle_scientific_notation(self):
             self.assertEqual(7, decimal_range.scale)
             self.assertEqual(2, decimal_range.precision)
 
+    def test_can_display_lower_is_upper_length(self):
+        self.assertEqual(six.text_type(ranges.DecimalRange("9.9...9.9")), "9.9...9.9")
+
+    def test_can_display_multi_range(self):
+        self.assertEqual(six.text_type(ranges.DecimalRange("9.1...10.9, 11...13")), "9.1...10.9, 11.0...13.0")
+
+    def test_can_represent_range_containing_none(self):
+        none_range = ranges.DecimalRange(None)
+        self.assertEqual(six.text_type(none_range), six.text_type(None))
+        upper_none_range = ranges.DecimalRange("1...")
```

Comment:
Which parts of the code is this test supposed to cover that are not already covered by the other tests? Adding items after the constructor should not be supported by `Range` and `DecimalRange`. Maybe we should make `items` a property that returns a `tuple` (that cannot be modified) instead of a `list`.